src/enemy.ts

```diff
@@ -12,8 +13,8 @@ export abstract class EnemyBase extends Phaser.GameObjects.Sprite {
 
     follower: any
     hp: integer
-    yOffset: number = Phaser.Math.RND.integerInRange(-20, 20);
-    xOffset: number = Phaser.Math.RND.integerInRange(-20, 20);
+    yOffset: number = PMath.RND.integerInRange(-20, 20);
```

Comment:
Did `Phaser.Math` stop working?

Reply:
yeah, it would just break in runtime, no idea why this works though